translate/storage/properties.py

```diff
@@ -380,6 +393,57 @@ class DialectGaia(DialectMozilla):
     delimiters = [u"="]
 
 
+@register_dialect
+class DialectGwt(DialectJava):
+    regex = re.compile(r'([^\[\]]*)(?:\[(.*)\])?')
+    name = "gwt"
+    default_encoding = "utf-8"
+    delimiters = [u"="]
+
+    gwt_plural_categories = [
+        ('', "other"),
+        ('none', "zero"),
+        ('one', 'one'),
+        ('two', 'two'),
+        ('few', 'few'),
+        ('many', 'many'),
+    ]
+
+    gwt2cldr = dict(gwt_plural_categories)
+    cldr2gwt = dict((v, k) for k, v in six.iteritems(gwt2cldr))
```

Comment:
I'm trying to create a bidirectionnal dict. in order to map gwt plural naming with cldr naming

Reply:
I understand that, but you already have the items list, so it might be better to use it directly:

```python
     cldr2gwt = dict((v, k) for k, v in gwt_plural_categories) 
````